### ISSUE TITLE
Data Preprocessing Concepts

### ISSUE THREAD

**Issue by chfor183** [chfor183/data_science_articles#12]
## TL;DR

1.	Load data
2.	Look at the data shape and type
3.	Drop duplicates
4.	Look at the few first and last rows
5.	Remove special characters from numeric columns
6.	Remove special characters from categorical columns
7.	Handling missing values
8.	Look at the data distribution
9.	Handling of outliers
10.	Dealing with skewed data
11.	Transformations if we use predictive modeling
        a.	Normalization
        b.	Scaling
        c.	Standardization
        d.	Balancing (oversampling / undersampling)
        e.	Binning
        f.	Categorical encoding

## Key Takeaways
- 1
- 2

## Useful Code Snippets
```
function test() {
  console.log("notice the blank line before this function?");
}
```

## Articles/Ressources
https://towardsdatascience.com/data-preprocessing-concepts-fa946d11c825
